### PR TITLE
[Translate]:Code-Style/2.3 Code Comments:P.CMT.[01, 02] && Change some details

### PR DESCRIPTION
**Pull request by Milittle**
Change some details.

### CODE REVIEW

File: src/safe-guides/code_style/comments/P.CMT.01.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+二、The documentation should be concise.
+
+1. The content and terms in the documentation comments should be as short and concise as possible, and should not be too long. Make sure your code is well commented and understandable by others. Good comments convey the context and purpose of the code.
+2. Comments are always structured around two key points:
+   - What: Used to describe what the code implement.
+   - How: Used to describe how to use code your code.
+3. The natural language used for comments and documentation comments should be consistent.
+4. Rust project documentation should always be built on the `rustdoc` tool. `rustdoc` supports the Markdown format. To make the documentation more beautiful and readable, the documentation comments should use the Markdown format.
+
+**[Examples]**
```
Comment (ZhangHanDong) [inline src/safe-guides/code_style/comments/P.CMT.01.md]: remove [Examples]

File: src/safe-guides/code_style/comments/P.CMT.01.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+
+**[Good]**
```
Comment (ZhangHanDong) [inline src/safe-guides/code_style/comments/P.CMT.01.md]: use [Good Case]

File: src/safe-guides/code_style/comments/P.CMT.02.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+
+**[Description]**
+
+The width of each comment line should not be too long and needs to be set at a certain width, no more than 120, which help improve readability.
+
+In `rustfmt`, the `comment_width` with `wrap_comments` configuration item can automatically split comments that exceed the width limit into multiple lines.
+
+**Note**: The `use_small_heuristics` configuration item of `rustfmt` does not include `comment_width`.
+
+**[Examples]**
```
Comment (ZhangHanDong) [inline src/safe-guides/code_style/comments/P.CMT.02.md]: remove [Examples]

File: src/safe-guides/code_style/comments/P.CMT.02.md [hunk 4]
```diff
@@ -1 +1,40 @@
...
+
+**[Bad]**
```
Comment (ZhangHanDong) [inline src/safe-guides/code_style/comments/P.CMT.02.md]: use  [Bad Case]

File: src/safe-guides/code_style/comments/P.CMT.02.md [hunk 5]
```diff
@@ -1 +1,40 @@
...
+
+```rust
+// Bad
+// Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+```
+
+**[Good]**
```
Comment (ZhangHanDong) [inline src/safe-guides/code_style/comments/P.CMT.02.md]: use [Good Case]

File: src/safe-guides/code_style/comments/P.CMT.01.md (hunk 1 above)
Comment (Milittle) [reply src/safe-guides/code_style/comments/P.CMT.01.md]: Done

File: src/safe-guides/code_style/comments/P.CMT.01.md (hunk 2 above)
Comment (Milittle) [reply src/safe-guides/code_style/comments/P.CMT.01.md]: Done

File: src/safe-guides/code_style/comments/P.CMT.02.md (hunk 3 above)
Comment (Milittle) [reply src/safe-guides/code_style/comments/P.CMT.02.md]: Done

File: src/safe-guides/code_style/comments/P.CMT.02.md (hunk 4 above)
Comment (Milittle) [reply src/safe-guides/code_style/comments/P.CMT.02.md]: Done

File: src/safe-guides/code_style/comments/P.CMT.02.md (hunk 5 above)
Comment (Milittle) [reply src/safe-guides/code_style/comments/P.CMT.02.md]: Done

**Pull request merged**